docker-bake.hcl

```diff
@@ -0,0 +1,110 @@
+# Documentation available at: https://docs.docker.com/build/bake/
+
+# Docker build args
+variable "DOCKER_REGISTRY" {default = "ghcr.io/openclarity"}
+variable "DOCKER_TAG" {default = "latest"}
+variable "SUFFIX" {default = ""}
+
+# Golang build args
+variable "VERSION" {default = null}
+variable "BUILD_TIMESTAMP" {default = null}
+variable "COMMIT_HASH" {default = null}
+variable "VMCLARITY_TOOLS_BASE" {default = null}
+
+function "get_tag" {
+  params = [name]
+  result = ["${DOCKER_REGISTRY}/${name}${SUFFIX}:${DOCKER_TAG}"]
+}
+
+group "default" {
+	targets = ["vmclarity-apiserver", "vmclarity-cli", "vmclarity-cr-discovery-server", "vmclarity-orchestrator", "vmclarity-ui", "vmclarity-ui-backend"]
+}
+
+group "e2e" {
+	targets = ["vmclarity-apiserver", "vmclarity-cli", "vmclarity-orchestrator", "vmclarity-ui", "vmclarity-ui-backend"]
```

Comment:
Same as above.

Reply:
Also do we really need this group? I think we should be ok with only having the `default` group. We do plan to extend E2E to be performed on K8s deployments as well.

Reply:
when I created the group, building the `cr-discovery-server` image was not yet part of the CI and e2e workflows, now it is not necessary indeed